lib/src/shim.dart

```diff
@@ -483,6 +509,11 @@ String _hoverMessage(HoverInformation hover) {
   return '$message';
 }
 
+String _hoverContentsPlaintext(HoverInformation hover) {
+  // TODO(dantup): Can we do better?
```

Comment:
I'm fine with this. I wouldn't bother with a separate method name, we can go with a comment that says we send markdown as "plaintext".

Reply:
I was gonna come back to this to see if we need to apply some basic transformations, but I'll tidy it up as if just plaintext for now so this PR is better and we can add it if/when required.

Reply:
I can see triple-backticks in the hovers, so I do think we can improve this - but I'll do it in a follow up PR if you're happy with this for now.